docs/k8s/README.md

```diff
@@ -1,6 +1,10 @@
 # Overview
 
-There are quite a few ways of running Tailscale inside a Kubernetes Cluster, some of the common ones are covered in this doc.
+There are quite a few ways of running Tailscale inside a Kubernetes Cluster.
+This doc covers creating and managing your own Tailscale node deployments in cluster.
+If you want a higher level of automation, easier configuration, automated tailscale device deletion or a mechanism for exposing the kube API server to the tailnet, take a look at [Tailscale Kubernetes operator](https://tailscale.com/kb/1236/kubernetes-operator).
+
+:warning: Note that the manifests generated by the following commands are not production ready Kubernetes deployment manifests and you will need to tweak them before actual usage. For example, the commands to generate a standalone proxy manifest, will create a standalone `Pod`- this will not persist accross cluster upgrades etc. :warning:
```

Comment:
We probably can't decide what "production ready" means for everyone, so I'd phrase this in terms of our intent or recommendations.

```suggestion
:warning: Note that the manifests generated by the following commands are not intended for production use, and you will need to tweak them based on your environment and use case. For example, the commands to generate a standalone proxy manifest, will create a standalone `Pod`- this will not persist across cluster upgrades etc. :warning:
```